tests/integration-async/records/indexer/test_literature.py

```diff
@@ -95,7 +93,7 @@ def test_index_record_manually(
 ):
     data = faker.record("lit")
     rec = LiteratureRecord.create(data)
-    models_committed.disconnect(index_after_commit)
+
     db.session.commit()
     models_committed.connect(index_after_commit)
```

Comment:
if you don't disconnect, you don't need to reconnect the signal. Is that correct though?

Reply:
I removed it by mistake, good catch